man/systemd-udevd.service.xml

```diff
@@ -105,6 +105,15 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>-s=</option></term>
```

Comment:
short options do not expect `=`

Reply:
`=` is used with all other short options in that man-page, but I guess that is wrong then. Should be fixed in separate PR